### ISSUE TITLE
Remove newburnaddress?

### ISSUE THREAD

**Issue by denravonska** [gridcoin-community/Gridcoin-Research#1226]
The `newburnaddress' RPC command generates a vanity burn address. It's fairly complicated and uses the [string/double/bin/hex](https://github.com/gridcoin-community/Gridcoin-Research/blame/886d9c0f7c60c5bdd99fa56b7044940fe5f7cdac/src/main.cpp#L2921) conversions which are slow and clunky.

While nice, the `newburnaddress` command does not seem that important. If we were to remove it we could with #1194 remove all those functions.

```
$ ./src/gridcoinresearchd newburnaddress
{
  "CombinedHex": "3e0368719ac5d3959985f476879a426bfff50c0c8a7aab9a70",
  "Burn Address": "RxGRCBurnAddressGRCBurnAddreuwHU1q"
}
```

**Comment by tomasbrod** [comment]
No. This will be useful for when we ditch hashBoinc in favor of OP_ERTURN

**Comment by denravonska** [comment]
@tomasbrod Why is it needed in that case?

**Comment by iFoggz** [comment]
OP_RETURN itself is not being removed so if we need some sort of rpc functionality with it it can be implemented better then the current newburnaddress rpc imo.

**Comment by denravonska** [comment]
Closed in favor of #1228
